### PR TITLE
Release v0.4.0 - Migrated the repository from Azure DevOps repos to GitHub

### PR DESCRIPTION
**Pull request by tforster**
- Bumped version number in package.json
- Ran `npm audit fix`
- Updated README.md for accuracy and added a DevOps Boards badge

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -94,20 +77,22 @@ The following is a list of the technologies used to develop and manage this proj
...
 | [NodeJS 12.3.0](https://nodejs.org/en/)                                                                           | Task running, automation and driving the API                                                         |
-| [NPM 6.10.1](https://www.npmjs.com/package/npm)                                                                   | Node package management                                                                              |
+| [NPM 6.14.5](https://www.npmjs.com/package/npm)                                                                   | Node package management                                                                              |
 | [Oh-My-Zsh](https://github.com/robbyrussell/oh-my-zsh)                                                            | ZSH shell enhancement                                                                                |
 | [Serverless Framework](https://serverless.com)                                                                    |                                                                                                      |
```
Comment (webdivelement) [inline README.md:85]: Noticed a missing description here.

File: package-lock.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@tforster/webproducer",
-  "version": "0.1.6",
+  "version": "1.0.0",
```
Comment (webdivelement) [inline package-lock.json:3]: Let's update the version number here to `0.4.0`.

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@tforster/webproducer",
-  "version": "0.1.6",
+  "version": "1.0.0",
```
Comment (webdivelement) [inline package.json:3]: Let's update the version number here to `0.4.0`.

File: README.md [hunk 4]
```diff
@@ -94,20 +77,22 @@ The following is a list of the technologies used to develop and manage this proj
...
 | [WSL 2](https://docs.microsoft.com/en-us/windows/wsl/install-win10)                                               | Windows Subsystem for Linux supports native Linux distributions                                      |
 | [ZSH](https://www.zsh.org/)                                                                                       | A better shell than Bash                                                                             |
 
 ## Change Log
 
+v1.0.0 **First milestone release** (2020-05-24)
+
 v0.1.6 **Improve stream stability** (2020-03-26)
 
 Bug fixes
```
[the remark this answers is not in the document]
Comment (webdivelement) [reply README.md]: Resolved after our initial chat.

File: README.md [hunk 5]
```diff
@@ -94,20 +77,22 @@ The following is a list of the technologies used to develop and manage this proj
...
+| [Ubuntu 20.04 for WSL2](https://www.microsoft.com/en-ca/p/ubuntu/9nblggh4msv6?activetab=pivot:overviewtab)        | Canonical supported Ubuntu for Windows Subsystem for Linux                                           |
+| [Visual Studio Code 1.45.1](https://code.visualstudio.com/)                                                       | Powerful and cross-platform code editor                                                              |
 | [Windows 10 Pro Insider Preview](https://www.microsoft.com/en-us/software-download/windowsinsiderpreviewadvanced) | The stable version of the Insiders build typically brings new tools of significant use to developers |
 | [WSL 2](https://docs.microsoft.com/en-us/windows/wsl/install-win10)                                               | Windows Subsystem for Linux supports native Linux distributions                                      |
 | [ZSH](https://www.zsh.org/)                                                                                       | A better shell than Bash                                                                             |
 
 ## Change Log
 
+v1.0.0 **First milestone release** (2020-05-24)
```
[the remark this answers is not in the document]
Comment (webdivelement) [reply README.md]: Resolved after our initial chat, will discuss versioning in greater detail during our next chat.

File: README.md [hunk 6]
```diff
@@ -1,5 +1,7 @@
 # WebProducer
 
+[![Board Status](https://dev.azure.com/techsmarts/08c518d9-553e-44a6-bd93-33b1b4b46b5c/94878bf3-1784-4a48-9a4a-e692d65425ba/_apis/work/boardbadge/e05cc4cb-70d4-4fcf-be68-9c1a6cb5cf69?columnOptions=1)](https://dev.azure.com/techsmarts/08c518d9-553e-44a6-bd93-33b1b4b46b5c/_boards/board/t/94878bf3-1784-4a48-9a4a-e692d65425ba/Microsoft.RequirementCategory/)
+
 WebProducer, or simply WP, is a serverless application for publishing content to the web. It equally supports websites and web applications with a lean, streams based, architecture.
 
 The current version is dependent upon the Amazon AWS stack, including Amplify, IAM, Lambda and S3. Support for other providers including Google and Microsoft is planned for the future.
```
Comment (tforster) [inline README.md:7]: Trying out suggested changes

File: README.md [hunk 7]
```diff
@@ -18,106 +20,89 @@ The core of the module is a NodeJS Lambda function. This function is intended to
 
 ## Installation
 
-WebProducer can be installed as a runtime dependency using your favourite package manager.
+WebProducer can be installed as a runtime dependency by adding to your package.json
```
Comment (tforster) [inline README.md:23]: Suggested change
```suggestion
 WebProducer Rocks
```

Comment (tforster) [reply README.md:23]: @webdivelement I'm not going to hit the "commit" button for obvious reasons but this is kinda cool. In the comment, I added a codefence with the suggestion keyword
 ```suggestion
new code I want to suggest goes here```

And it is now offered as a single click to implement to the PR author.

**Pull request closed without merging**